src/router/routes.js

```diff
@@ -28,6 +28,7 @@ export default [
 	{ path: '/lend/filter', component: () => import('@/pages/Lend/Filter/LendFilterPage') },
 	{ path: '/page-two', component: () => import('@/pages/PageTwo') },
 	{ path: '/portfolio/lending-stats', component: () => import('@/pages/LendingStats/LendingStatsPage') },
+	{ path: '/thanks', component: () => import('@/pages/Thanks/Thanks') },
```

Comment:
I think this is good for now as we won't be ready for launch this sprint... we'll need the .htaccess entry too.

@emuvente, do you have any thoughts on if we are even able to use the existing path `/basket/thanks` when we have 2 versions of thanks... seems like our current proxy setup won't allow use to use that path in both Ui and Php...

Reply:
I added the .htaccess entry to this other code review: https://github.com/kiva/kiva/pull/8158

Reply:
Updated my comment above.

Reply:
with my latest commit I've updated the route for the new thanks page to be /checkout/thanks since /thanks is taken by our existing thanks page.